.github/fabricbot.json

```diff
@@ -12,106 +12,99 @@
             "area-System.Security"
           ],
           "mentionees": [
-            "bartonjs",
+            "dotnet/area-system-security",
             "vcsjones",
-            "krwq",
-            "GrabYourPitchForks"
+            "krwq"
           ]
         },
         {
           "labels": [
             "area-System.Drawing"
           ],
           "mentionees": [
-            "safern",
-            "tarekgh"
+            "dotnet/area-system-drawing"
           ]
         },
         {
           "labels": [
             "area-System.Diagnostics.Tracing"
           ],
           "mentionees": [
-            "tarekgh",
-            "tommcdon",
-            "pjanotti"
+            "dotnet/area-system-diagnostics-tracing"
           ]
         },
         {
           "labels": [
             "area-System.Globalization"
           ],
           "mentionees": [
-            "tarekgh",
-            "safern"
+            "dotnet/area-system-globalization"
           ]
         },
         {
           "labels": [
             "area-System.Linq.Parallel"
           ],
           "mentionees": [
-            "tarekgh",
-            "carlossanlop"
+            "dotnet/area-system-linq-parallel"
           ]
         },
         {
           "labels": [
             "area-System.Resources"
           ],
           "mentionees": [
-            "tarekgh",
-            "buyaa-n"
+            "dotnet/area-system-resources"
           ]
         },
         {
           "labels": [
             "area-System.Text.Encoding"
           ],
           "mentionees": [
-            "tarekgh",
-            "dotnet/area-system-text-encoding"
+            "dotnet/area-system-text-encoding",
+            "tarekgh"
           ]
         },
         {
           "labels": [
             "area-System.Text.Encodings.Web"
           ],
           "mentionees": [
-            "tarekgh",
-            "dotnet/area-system-text-encodings-web"
+            "dotnet/area-system-text-encodings-web",
+            "tarekgh"
```

Comment:
I've added Tarek to this team so we can remove his explicit inclusion.

Reply:
FWIW we might want to make similar changes in the dotnet/runtime FB configs

Reply:
I'll spin up a PR over there.

Reply:
Done: https://github.com/dotnet/runtime/pull/64199